website/docs/docs/build/packages.md

```diff
@@ -173,19 +173,37 @@ To use native private packages, you must have one of the following Git providers
 
 #### Configuration
 
-Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable:
+Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable. 
+
+The private key supports a two-layered-path: `org/repo`. dbt inherits the project path from your source repository.
 
 <File name="packages.yml">
 
 ```yaml
 packages:
-  - private: dbt-labs/awesome_repo
+  - private: dbt-labs/awesome_repo # your-org/your-repo path
   - package: normal packages
-
-	[...]
+  [...]
 ```
+</File>
+
+:::tip Azure DevOps repositories
```

Comment:
Maybe this should be titled "Azure DevOps limitations?"

Reply:
that's a good flag! i usually try use more positive language and tend to use considerations more than limitations. would Azure DevOps considerations work?

Reply:
![Screenshot 2024-12-24 at 10 04 24](https://github.com/user-attachments/assets/ff6f1149-c224-4784-a38d-b52ae302834f)